src/ui_basic/window.cc

```diff
@@ -139,7 +117,7 @@ Window::Window(Panel* const parent,
                             kWindowTitlebarButtonsSize,
                             kWindowTitlebarButtonsSize,
                             ButtonStyle::kWuiSecondary,
-                            g_image_cache->get(window_image_path(kWindowImageUnpinned)),
+                            "",
```

Comment:
Would it be safer to initialize this with an actual image from the style? Same below.

Reply:
The constructor calls `update_toolbar_buttons()` a few lines below were the correct image and tooltip are set. No need to make this redundant imho.